src/common/domain/Journal.d.ts

```diff
@@ -0,0 +1,89 @@
+/* tslint:disable */
```

Comment:
Fine for now to get us going, but once we depend on the journal type package this will need to be deleted.